src/Experiment.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Should this be 6.0?

Reply:
Or NetCoreAppCurrent?

Reply:
If we wanted to do this I would have to add a workaround for the 5.0 SDK to not complain about a not supported TFM, so that's why I decided to leave with net5.0 for now.

Reply:
NetCoreAppCurrent doesn't exist on this branch. We could add it though, but given the small number of projects I would imagine a branch would have I don't know if it makes sense.